deployments/docker-compose.tezos.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.2"
+
+volumes:
+  node_data:
+  client_data:
+
+services:
+ tezosnode:
+    image: tezos/tezos:mainnet
+    hostname: tezosnode
+    command: tezos-node --net-addr :9732
+    ports:
+      - "9732:9732"
+      - "8732:8732"
+    expose:
+      - "8732"
+    volumes:
```

Comment:
we prob don't want to expose the ports, the client can access them on the scrapers-network

Reply:
Yes. I will update so it does not try to update if it fails to retrieve the block